python/semantic_kernel/agents/agent.py

```diff
@@ -293,18 +299,24 @@ def invoke_stream(
         *,
         messages: str | ChatMessageContent | list[str | ChatMessageContent] | None = None,
         thread: AgentThread | None = None,
+        on_intermediate_message: Callable[[ChatMessageContent], Awaitable[None]] | None = None,
```

Comment:
Now that `OpenAIResponsesAgent` is in main, we'll need to update its invoke methods to reflect this change.